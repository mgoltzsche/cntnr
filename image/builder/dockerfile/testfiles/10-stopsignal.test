FROM alpine:3.7
STOPSIGNAL 9

# ASSERT CFG /config/StopSignal=9