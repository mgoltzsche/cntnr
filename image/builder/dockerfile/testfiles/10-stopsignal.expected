FROM alpine:3.7
STOPSIGNAL 9